### PR TITLE
Support for Laravel 6.0 and Sweet-Alert 2.0

### PR DESCRIPTION
**Pull request by backstageel**
This PR only adds support for Laravel 6.0 and Sweet Alert 2.0

### CODE REVIEW

**Review by decalages — APPROVED** [review]

**Review by nasrulhazim — APPROVED** [review]
This should work with Laravel 6.0

**Comment by nasrulhazim** [comment]
temporarily I added the following and update my sweet alert.

```
    "repositories": [{
        "type": "vcs",
        "url": "https://github.com/backstageel/sweet-alert"
    }]
```

**Comment by anburocky3** [comment]
> vcs

Here is the reference file: https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository

After adding your snippet, what to do? Need to insert this?
 {
        "uxweb/sweet-alert": "dev-bugfix"
    }

**Comment by decalages** [comment]
Hi @anburocky3,
I updated my composer.json file as follows:
```
...
 "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/backstageel/sweet-alert"
        }
    ],
    "require": {
        "php": "^7.2",
        "fideloper/proxy": "^4.0",
        "laravel/framework": "^6.0",
        "laravel/tinker": "^1.0",
        "uxweb/sweet-alert": "^2.0"
    },
...
```

File: src/SweetAlert/SweetAlertNotifier.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ public function message($text = '', $title = null, $icon = null)
         }
 
         if (!is_null($icon)) {
-            $this->config['icon'] = $icon;
+            $this->config['type'] = $icon;
```
Comment (anburocky3) [inline src/SweetAlert/SweetAlertNotifier.php:82]: Should it be replaced from **`$this->config['type'] = $icon;`** to **`$this->config['icon'] = $icon;`**

Comment (backstageel) [reply src/SweetAlert/SweetAlertNotifier.php:82]: sweet-alert2 uses 'type' instead of 'icon'

Comment (anburocky3) [reply src/SweetAlert/SweetAlertNotifier.php:82]: I don't why it is not working.
For me `$this->config['icon'] = $icon;` only working

Comment (HazemAyyad) [reply src/SweetAlert/SweetAlertNotifier.php:82]: this change apply just laravel 6
Should it be replaced 
from     $this->config['icon'] = $icon; 
to          $this->config['type'] = $icon;

**Pull request merged**